index.html

```diff
@@ -14,14 +14,83 @@
     <link rel="stylesheet" href="css/reset.css" />
     <link rel="stylesheet" href="css/layout.css" />
     <link rel="stylesheet" href="css/styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Proza+Libre:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap"
+      rel="stylesheet" />
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="header-bar">
+        <div class="container">
+          <div class="row">
+            <div class="column-full">
+              <div class="header-title">
+                <p>Code Journal</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
-      <div data-view="entries" class="hidden"></div>
+      <form id="entry-form">
+        <div data-view="entry-form">
+          <div class="row">
+            <div class="column-full">
+              <h1>New Entry</h1>
+            </div>
+          </div>
+          <div class="row" id="img-inputs">
+            <div class="column-half">
+              <div id="img-wrapper">
```

Comment:
This `img-wrapper` isn't needed. Just have the `img` be a direct child to `column-half` and give your `img` `width: 100%`